packages/now-bash/index.js

```diff
@@ -12,9 +12,10 @@ exports.config = {
 
 exports.analyze = ({ files, entrypoint }) => files[entrypoint].digest;
 
-exports.build = async ({ files, entrypoint, config }) => {
+exports.build = async ({
+  workPath, files, entrypoint, config,
+}) => {
   const srcDir = await getWritableDirectory();
```

Comment:
```suggestion
  const srcDir = await getWritableDirectory();
  workPath = workPath || await getWritableDirectory();
```

I think you need to keep this backwards compatible